### PR TITLE
[node] Remove need for tsc in Node

### PR DESCRIPTION
**Pull request by snario**
Don't think we need this

### CODE REVIEW

File: packages/node/rollup.config.js [hunk 1]
```diff
@@ -62,9 +62,7 @@ export default [
       nodeResolve({
         only: [...bundledDependencies]
       }),
-      typescript({
-        tsconfig: "tsconfig.rollup.json"
-      }),
+      typescript(),
```
Comment (cf19drofxots) [inline packages/node/rollup.config.js]: If the `tsconfig.json` is removed then what configuration is rollup looking at?

Comment (cf19drofxots) [reply packages/node/rollup.config.js]: 👍

**Comment by cf19drofxots** [comment]
How does this affect an IDE's type checking that's based on a local installation of `tsc`?

**Comment by snario** [comment]
> How does this affect an IDE's type checking that's based on a local installation of `tsc`?

Shouldn't have an effect

**Comment by cf19drofxots** [comment]
@snario this breaks transpilation for the tests so a `yarn build` doesn't catch errors anymore.

Not sure if this optimization is worth the hassle of figuring out a workaround. The previous configuration was working just fine.

**Comment by snario** [comment]
> @snario this breaks transpilation for the tests so a `yarn build` doesn't catch errors anymore.
> 
> Not sure if this optimization is worth the hassle of figuring out a workaround. The previous configuration was working just fine.

https://github.com/counterfactual/monorepo/pull/527

**Pull request merged**